Getting-Started.md

```diff
@@ -37,6 +37,87 @@ You will need to install ElasticSearch (6.1.0), Kibiter (6.1.4) and a MySQL/Mari
 You will have to install the GrimoireLab components listed above, and use the following docker-compose to have ElasticSearch, Kibiter and MariaDB.
 Note that you can omit the `mariadb` section in case you have MySQL/MariaDB already installed in your system.
 
+***
+
+#### Docker installation
+
+Installation for Ubuntu 
+
+[Reference for Docker installation](https://phoenixnap.com/kb/how-to-install-docker-on-ubuntu-18-04)
+
+```
+sudo apt-get update
+```
+```
+sudo apt install docker.io
+```
+![docker_in1](https://user-images.githubusercontent.com/53489999/77804454-8d414080-70a5-11ea-9d47-1e18541c885f.png)
+
+```
+sudo systemctl start docker
+sudo systemctl enable docker
+```
+![docker_in2](https://user-images.githubusercontent.com/53489999/77804492-a8ac4b80-70a5-11ea-8d36-b92abba57404.png)
+
+To check whether docker has installed correctly.
+
+```
+Docker --version 
+```
+
+![docker_version_check](https://user-images.githubusercontent.com/53489999/77804520-bcf04880-70a5-11ea-88e1-c439a872f65f.png)
+
+Add user 
+```
+sudo gpasswd -a $USER docker
+```
+Create a group
+
+```
+newgrp docker
+```
+
+Run your first image (hello-world)
+
+> If the image is not available locally - docker gets the image from dockerhub.
+
+```
+docker run hello-world
+```
+![docker-hello_world_image](https://user-images.githubusercontent.com/53489999/77804566-e4471580-70a5-11ea-9e07-0daed701706a.png)
+
+***
+
+#### Docker-compose installation
+
+Select Linux  at 
+[Install Docker compose](https://docs.docker.com/compose/install/)
+
+
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.25.4/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+```
+
+```
+sudo chmod +x /usr/local/bin/docker-compose
+
+docker-compose --version
+```
+
+![docker-compose1](https://user-images.githubusercontent.com/53489999/77804770-5586c880-70a6-11ea-9972-5bfef57d8f4c.png)
+
+```
+docker-compose ps
+
+docker-compose ps -a
+
+docker-stop <container-id>
+```
+
+![docker-compose2](https://user-images.githubusercontent.com/53489999/77804808-70593d00-70a6-11ea-9370-201e26c73e52.png)
+
+***
+
```

Comment:
Also, I have a small suggestion. The whole explanation is about installing docker and docker-compose. How about just having simple links to the original docker docs like
- https://docs.docker.com/install/linux/docker-ce/ubuntu/
- https://docs.docker.com/compose/install/

or maybe to link to some blog post which explains the installation really well along with pictures.

The thing is, I am just worried about the length. :sweat_smile: 

This suggestion is arguable. So, I would like to know your views.
@pugnator-12 @valeriocos, WDYT?

Reply:
+1 for the links to the original docker docs

> The thing is, I am just worried about the length.

Yes, indeed. As soon as there is enough material, I'd like to move all the getting started content to the tutorial repository or wrap it as a user guide. WDYT about the plan?

Reply:
Sorry for barging in. Couldn't stop cause the plan is great @valeriocos. This is indeed becoming quite lengthy.

Reply:
Sure.. we can do that, its a good idea.